app/bundles/CampaignBundle/Entity/Event.php

```diff
@@ -134,6 +134,14 @@ class Event
      */
     private $changes;
 
+    /** CAPTIVEA.CORE START **/
+    /**
+     *
+     * @var \Mautic\ScoringBundle\Entity\ScoringCategory 
+     */
+    private $scoringCategory = null;
+    /** CAPTIVEA.CORE END **/
```

Comment:
There is really no need for personal comments. Otherwise Mautic would be full of them. Lets use git to track who is author of what line.